x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/component/scan_usage_results.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiCheckbox,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiText,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import React from 'react';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { css } from '@emotion/react';
+import { InferenceUsageInfo } from '../../../../types';
+import { useKibana } from '../../../../../../hooks/use_kibana';
+import { RenderMessageWithIcon } from './render_message_with_icon';
+
+import * as i18n from '../delete/confirm_delete_endpoint/translations';
+import { ListUsageResults } from './list_usage_results';
+
+interface ScanUsageResultsProps {
+  list: InferenceUsageInfo[];
+  ignoreWarningCheckbox: boolean;
+  onCheckboxChange: (state: boolean) => void;
+}
+
+export const ScanUsageResults: React.FC<ScanUsageResultsProps> = ({
+  list,
+  ignoreWarningCheckbox,
+  onCheckboxChange,
+}) => {
+  const {
+    services: { application },
+  } = useKibana();
+  const handleNavigateToIndex = () => {
```

Comment:
nitpick - nonblocking: It should be named as `handleNavigateToIndexManagement` rather